features/steps/steps.rb

```diff
@@ -0,0 +1,69 @@
+Then("the Content-Type header is valid multipart form-data") do
+  expected = /^multipart\/form-data; boundary=--------------------------\d+$/
+  actual = Server.current_request[:request]["content-type"]
+
+  assert_match(expected, actual)
+end
+
+def read_expected_file(fixture, file_name)
+  path = "#{__dir__}/../expected/#{fixture}/#{file_name}"
+
+  assert_true(File.exists?(path), "The file '#{file_name}' does not exist at '#{path}'")
+
+  File.read(path)
+end
+
+def get_form_data_as_string(field)
```

Comment:
I'm not sure we'd need this specifically, as we've been able to access multipart elements as strings previously [as seen here](https://github.com/bugsnag/maze-runner/blob/master/lib/features/steps/request_assertion_steps.rb#L119).

Does that not work for this kind of multi-part request?